ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -155,23 +151,24 @@ public static void main(String args[]) throws Exception {
       }
       Time time = SystemTime.getInstance();
       Throttler throttler = new Throttler(config.indexEntriesToProcessPerSec, 1000, true, time);
-      StoreKeyConverterFactory storeKeyConverterFactory = Utils.getObj(serverConfig.serverStoreKeyConverterFactory, properties, clusterMap.getMetricRegistry());
+      StoreKeyConverterFactory storeKeyConverterFactory =
+          Utils.getObj(serverConfig.serverStoreKeyConverterFactory, properties, clusterMap.getMetricRegistry());
       ConsistencyCheckerTool consistencyCheckerTool =
-          new ConsistencyCheckerTool(clusterMap, blobIdFactory, storeConfig, filterKeySet, throttler, metrics, time, storeKeyConverterFactory.getStoreKeyConverter());
+          new ConsistencyCheckerTool(clusterMap, blobIdFactory, storeConfig, filterKeySet, throttler, metrics, time,
+              storeKeyConverterFactory.getStoreKeyConverter());
       boolean success =
           consistencyCheckerTool.checkConsistency(config.pathOfInput.listFiles(File::isDirectory)).getFirst();
       System.exit(success ? 0 : 1);
     }
   }
 
   public ConsistencyCheckerTool(ClusterMap clusterMap, StoreKeyFactory storeKeyFactory, StoreConfig storeConfig,
```

Comment:
minor: can give this an instance of `IndexProcessingHelper` instead (since this class has no extra input to provide for its construction) which can help share instances across multiple components that may need the `IndexProcessingHelper`

Reply:
See suggestions to drop `IndexProcessingHelper` altogether